mandatory/4-tax.js

```diff
@@ -5,7 +5,11 @@
   Sales tax is 20% of the price of the product.
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(price) {
+ return price + price*0.2;
```

Comment:
Change to this option? 
function calculateSalesTax(sales) {
  let tax = sales * 0.2;
  let total = sales + tax;
  return total;
}